CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Changes
 =======
 
+3.4b1 (2019-08-17)
+------------------
+
+* Suport using :ref:`chromium <arg-engine>` instead of webkit as the
+  rendering engine. It is in pre-alpha stage so not suggested for
+  using in production projects.
+* Qt is updaeted to 5.13, and QtWebkit is updated to
+  QtWebkit-5.212.3-alpha3.
+* Twisted is updated to 19.7.0.
+* Use ubuntu:18.04 as the base docker image.
+* HTTP2 can be disabled with
```

Comment:
the important thing is that Splash got http2 support, which it haven't had before